<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>216</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Currency Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="outputBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="inputCurrency">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="outputCurrency">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="swapButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/swap-vertical.png</normaloff>../../Downloads/swap-vertical.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="to">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QLabel" name="convert">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
